src/main/scala/wdltool/graph/WomGraphPrint.scala

```diff
@@ -0,0 +1,130 @@
+package wdltool.graph
+
+import java.nio.file.{Files, Paths}
+import java.util.concurrent.atomic.AtomicInteger
+
+import better.files.File
+import cats.data.Validated.{Invalid, Valid}
+import cats.derived.monoid._
+import cats.derived.monoid.legacy._
+import cats.implicits._
+import wdl4s.cwl.{CommandLineTool, CwlCodecs, Workflow, Yaml}
+import wdl4s.wdl.{WdlNamespace, WdlNamespaceWithWorkflow}
+import wdl4s.wom.executable.Executable
+import wdl4s.wom.graph.GraphNodePort.{InputPort, OutputPort}
+import wdl4s.wom.graph.{GraphInputNode, GraphNodePort, GraphOutputNode}
+import wdltool.graph.GraphPrint.clusterCount
+//import wdl4s.wdl.{CallOutput, Declaration, If, Scatter, WdlCall, WdlGraphNode, _}
+import wdl4s.wom.graph.{CallNode, Graph, GraphNode, ScatterNode}
+
+import scala.collection.JavaConverters._
+
+object WomGraphPrint {
+
+  final case class WorkflowDigraph(workflowName: String, digraph: NodesAndLinks)
+  final case class NodesAndLinks(nodes: Set[String], links: Set[String])
+
+  def readFile(filePath: String): String = Files.readAllLines(Paths.get(filePath)).asScala.mkString(System.lineSeparator())
+
+  def womExecutableFromWdl(filePath: String): Executable = {
+    val namespace = WdlNamespaceWithWorkflow.load(readFile(filePath), Seq(WdlNamespace.fileResolver _)).get
+    namespace.womExecutable
+  }
+
+  def womExecutableFromCwl(filePath: String): Executable = {
+    val yaml: Yaml = readFile(filePath)
+    CwlCodecs.decodeCwl(yaml) match {
+      case Right((clt: CommandLineTool, _)) =>
+        clt.womExecutable match {
+          case Valid(wom) => wom
+        }
+      case Right((wf: Workflow, m)) if m.isEmpty =>
+        wf.womExecutable(Map.empty) match {
+          case Valid(wom) => wom
+        }
+        // TODO: Don't throw away the error information! Might as well return it:
+      case other => throw new Exception(s"Can't read $filePath")
+    }
```

Comment:
ok so all these lines of boilerplate are somewhat suggestive of the need for a friendlier CWL API...